Clarify what happens when no bootnodes are specified. (#1477)
hyperledger/besu-docs: versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md

## Configure bootnodes in a production network

> A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. In case of no bootnodes, Mainnet default bootnodes will be used.

Reviewer: ```suggestion
A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Mainnet default bootnodes will be used.
```

> A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Mainnet default bootnodes will be used.

Reviewer 2: ```suggestion
A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Besu uses Mainnet's default bootnodes.
```

Author: Done! Thanks @alexandratran
